test/indexes.test.js

```diff
@@ -120,107 +120,108 @@ describe('Indexes', function () {
       assert.deepEqual(idx.tree.search('world'), []);
       assert.deepEqual(idx.tree.search('bloup'), []);
     });
-	
```

Comment:
I was coding on Windows at that time (if you can believe it), and Windows gvim is not whitespace-friendly :)